### ISSUE TITLE
[Bug]: I use console-ninja not working 

### ISSUE THREAD

**Issue by boyThanawatKos** [wallabyjs/console-ninja#337]
### What happened?

Now using console-ninja Community, it worked fine before, but now it can't display output, even simple tests. Sorry, I'm a newbie and may not know much.

I use next.js v.14.2.0
![trace-log](https://github.com/user-attachments/assets/a0c6b9a9-c96a-4a2b-a9b8-83af122a36de)

### Version

v1.0.357

### CLI command to start your dev tool

yarn dev

### Steps to reproduce (or sample repo)

I try console type server side and client side so much but can't show display output 

### Log output

```shell
03:20:20.457 info host client connected: buildHook { address: '127.0.0.1', family: 'IPv4', port: 51696 }
03:20:20.461 WARN host ✘ next.js v14.2.7 is not yet supported in the Community edition of Console Ninja.
03:20:20.461 WARN host We are working hard on it for you https://tinyurl.com/3h9mtwra.
03:20:20.461 WARN host 
03:20:20.461 WARN host Estimated release dates:
03:20:20.461 WARN host   - Community users: around 27th October, 2024 (subject to team availability)
03:20:20.461 WARN host   - PRO users:       priority access is available now
03:20:20.917 info buildHook-51045 installing build hook for next.js at /volumes/disk d/nakamoto_files/frontend_refactor, node v20.2.0
03:20:20.968 info host client connected: buildHook { address: '127.0.0.1', family: 'IPv4', port: 51696 }
03:20:20.974 WARN host ✘ next.js v14.2.7 is not yet supported in the Community edition of Console Ninja.
03:20:20.974 WARN host We are working hard on it for you https://tinyurl.com/3h9mtwra.
03:20:20.974 WARN host 
03:20:20.974 WARN host Estimated release dates:
03:20:20.974 WARN host   - Community users: around 27th October, 2024 (subject to team availability)
03:20:20.974 WARN host   - PRO users:       priority access is available now
```

**Comment by smcenlly** [comment]
Your screenshot and log show that you are using a new version of `next.js` that is not yet supported in the Community Edition. It should be available round 27th October, 2024.

You may use an earlier version of `next.js` that is supported by the Community Edition.

For those using [Console Ninja PRO edition](https://console-ninja.com/pro), next@14.2.7 priority access is already available.

**Comment by boyThanawatKos** [comment]
ok i can do it, i fix version in package json "next": "14.2.0" , before i use ^14.2.0 

**THANK YOU SO MUCH**

**Comment by boyThanawatKos** [comment]
> Your screenshot and log show that you are using a new version of `next.js` that is not yet supported in the Community Edition. It should be available round 27th October, 2024.
> 
> You may use an earlier version of `next.js` that is supported by the Community Edition.
> 
> For those using [Console Ninja PRO edition](https://console-ninja.com/pro), next@14.2.7 priority access is already available.

ok i can do it, i fix version in package json "next": "14.2.0" , before i use ^14.2.0 

**THANK YOU SO MUCH**
